drivers/storage/portworx/component/telemetry.go

```diff
@@ -668,7 +681,7 @@ func (t *telemetry) createConfigMap(
         "use_appliance_id": "true"
       },
       "proxy": {
-        "path": "/dev/null"
+        "path": "/cache/network/http_proxy"
```

Comment:
nit: We should use the constant `TelemetryCCMProxyFilePath` here too.

Reply:
the whole ccm property config is a const string, I feel there's no point to just change this field only..